parts/linux/cloud-init/artifacts/components.json

```diff
@@ -431,6 +435,10 @@
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes-csi/csi-node-driver-registrar",
           "latestVersion": "v2.12.0"
```

Comment:
can we remove 2.12.0/2.14.0?

Reply:
@cameronmeissner these two versions are still used in aks-rp, after we have upgraded to new versions, then I will remove the old ones